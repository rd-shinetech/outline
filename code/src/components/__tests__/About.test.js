import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import About from '../About';


describe('About.test.js', () => {

    it('renders as expected', () => {
        const wrapper = shallow(<About/>);

        expect(wrapper.length).toEqual(1);

        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();
    });
});
