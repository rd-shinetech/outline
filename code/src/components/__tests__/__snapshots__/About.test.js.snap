exports[`About.test.js renders as expected 1`] = `
<div
  className="container">
  <h1>
    About
  </h1>
  <p>
    Est et amet perfecto sententiae, nec error essent eripuit ei. Velit sanctus ut has, partem dolorem atomorum est ad, sumo fabellas electram ex vim.
  </p>
</div>
`;
